lib/browser/api/screen.ts

```diff
@@ -1,38 +1,41 @@
+import { EventEmitter } from 'events';
+
 const { createScreen } = process._linkedBinding('electron_common_screen');
 
 let _screen: Electron.Screen;
 
+const ensureScreen = () => {
+  if (_screen === undefined) {
+    _screen = createScreen();
+  }
+};
+
 // We can't call createScreen until after app.on('ready'), but this module
 // exposes an instance created by createScreen. In order to avoid
 // side-effecting and calling createScreen upon import of this module, instead
 // we export a proxy which lazily calls createScreen on first access.
 export default new Proxy({}, {
   get: (target, prop: keyof Electron.Screen) => {
-    if (_screen === undefined) {
-      _screen = createScreen();
-    }
+    ensureScreen();
     const v = _screen[prop];
     if (typeof v === 'function') {
       return v.bind(_screen);
     }
     return v;
   },
   ownKeys: () => {
-    if (_screen === undefined) {
-      _screen = createScreen();
-    }
+    ensureScreen();
     return Reflect.ownKeys(_screen);
   },
-  has: (target, prop: string) => {
-    if (_screen === undefined) {
-      _screen = createScreen();
-    }
+  getPrototypeOf: () => {
+    return EventEmitter.prototype;
```

Comment:
Is this 100% accurate? I would have thought this would be
```suggestion
    if (_screen === undefined) {
      _screen = createScreen();
    }
    return Reflect.getPrototypeOf(_screen);
```

Reply:
@MarshallOfSound that prototype is always null - it's some weirdness with the c++ prototype setters